tests/tests.rs

```diff
@@ -93,6 +93,76 @@ fn converts_floats_to_i32() {
     assert_eq!(gen_i32(), 3);
 }
 
+#[test]
+fn converts_vec_floats_to_i32() {
+    #[replace_float_literals(literal as i32)]
+    fn gen_i32_vec_1() -> Vec<i32> {
+        vec![3.2, 5.7, 10.1]
+    }
+
+    assert_eq!(gen_i32_vec_1(), vec![3, 5, 10]);
+
+    #[replace_float_literals(literal as i32)]
+    fn gen_i32_vec_2() -> Vec<i32> {
+        vec![2.99; 4]
+    }
+
+    assert_eq!(gen_i32_vec_2(), vec![2, 2, 2, 2]);
+}
+
+#[test]
+fn converts_vec_integers_to_f64() {
+    #[replace_int_literals(literal as f64)]
+    fn gen_f64_vec() -> Vec<f64> {
+        vec![3 / 2, 1 / 2, 9 / 2]
+    }
+
+    assert_eq!(gen_f64_vec(), vec![1.5, 0.5, 4.5]);
+}
+
+#[test]
+fn converts_vec_numeric() {
+    #[replace_numeric_literals(literal as i32)]
+    fn gen_i32_vec() -> Vec<i32> {
+        vec![3.2, 5.7, 10.1]
+    }
+
+    assert_eq!(gen_i32_vec(), vec![3, 5, 10]);
+
+    #[replace_numeric_literals(literal as f64)]
+    fn gen_f64_vec() -> Vec<f64> {
+        vec![3 / 2, 1 / 2, 9 / 2]
+    }
+
+    assert_eq!(gen_f64_vec(), vec![1.5, 0.5, 4.5]);
+}
+
+#[test]
+fn converts_assert_eq_floats_to_i32() {
+    #[replace_float_literals(literal as i32)]
+    fn assert_eq_test() -> bool {
```

Comment:
Is there a reason for not just having a function without a return value here?

Reply:
No, I guess returning something here is completely  unnecessary.

Reply:
Fixed